### DISCUSSION TITLE
gt trajectory

### DISCUSSION THREAD

**Discussion by chengwei0427** [DapengFeng/S3E#22] [Q&A] [answered]
Hi, @DapengFeng 
Nice work.
Where can I find the gt of the trajectory?

**Comment by DapengFeng** [comment, marked as answer]
S3E is well organized under the following structure. <agent_id>_gt.txt is the gt trajectory for each agent in the sequence.
![Screenshot from 2024-08-28 20-40-34](https://github.com/user-attachments/assets/fe8afcf8-5395-4488-9f6a-cdb4114503f4)


You can find the gt trajectory at https://huggingface.co/datasets/PengYu-Team/S3E/tree/main

**Reply by chengwei0427** [reply]
![image](https://github.com/user-attachments/assets/b82ea386-2666-41f4-b505-4540231c29a8)
I cannot find the gt trajectory?

**Reply by DapengFeng** [reply]
In laboratories and teaching buildings sequences , restricted to the GNSS signal, we only record the start point and end point of each agent. For example, in laboratories, we record them using the Vicon; in teaching buildings, we record them in the GNSS available regions. For other sequences, we recorded the whole trajectory using the GNSS.

**Reply by chengwei0427** [reply]
Get √.
Thank you very much.
